src/coreclr/jit/gentree.h

```diff
@@ -2224,6 +2230,41 @@ struct GenTree
         return (gtOper == GT_CNS_INT) ? (gtFlags & GTF_ICON_HDL_MASK) : GTF_EMPTY;
     }
 
+#ifdef FEATURE_HW_INTRINSICS
+
+    void ClearEmbRoundingMode()
+    {
+        assert(gtOper == GT_HWINTRINSIC);
+        gtFlags &= ~GTF_HW_ER_MASK;
+    }
+    void SetEmbRoundingMode(uint8_t mode)
```

Comment:
Can you please add function comments? In particular, it would be nice to know that `mode` is one of the values from System.Runtime.Intrinsics.X86.FloatRoundingMode.

Reply:
I think it is fine to go with no comma.

we adopted the no-comma format in embedded broadcast (#90123), and also [MSVC](https://godbolt.org/z/dsjxvox9s) adopted the same format.